.github/workflows/sync-opencraft-release-palm-1.yml

```diff
@@ -0,0 +1,50 @@
+name: Sync opencraft-release/palm.1 with upstream
```

Comment:
Could we call this repo `.github` (like [this one](https://github.com/openedx/.github))? We can use it for more automations than branch syncing in the future.